lib/modules/common/exceptions.ts

```diff
@@ -0,0 +1,3 @@
+export const exceptions = {
+    AUTH_BYPASS_ERROR: "Configuration Error: bypassAccountId must be set if authentication bypass is in effect",
+};
```

Comment:
@kurtisauc3 would you rather have a top level file to hold exceptions for all modules, controllers, etc or does it make sense for this to live here?